### ISSUE TITLE
RSTP push for IPcam behind a router without a public IP

### ISSUE THREAD

**Issue by andreap-** [roleoroleo/yi-hack-MStar#229]
It may happen you need to place your IPcamera somewhere without the possibilty to forward a port on the router, or your provider just does not give you a public IP
EX you're on holiday in an Hotel and you want to leave your baby sleeping in the room while you're in the hall for your breakfast.
In bigger Hotels (or with a more sophisticated networking) the 2 areas may be on different lan unable do comunicate with each other.
So the idea is you have a RSTP server at home/office/private_cloud you use to relay you RSTP stream
basically it's the same problem you have with remote assistance software.
I stumbled upon this RSTP PUSH and I think it should do the trick
plus it won't take room in the little memory available on this IPcams
Do you think you can develop it
http://www.happytimesoft.com/howto/how-to-setup-rtsp-pusher.html

**Comment by stingerlu** [comment]
Not sure if this will work for you, but I have solved it by doing reverse tunneling to my ssh server, using the dropbearmulti binary.

Something like:

/home/yi-hack/bin/ssh -y -y -R localhost:<destination_port>:0.0.0.0:22 <user>@<sshd_host> -p <sshd_port> -i <id_rsa>
